src/listen.js

```diff
@@ -152,6 +152,23 @@ module.exports = function(defaultFuncs, api, ctx) {
                 })(0)
                 break;
               }
+              
+              if (v.delta.class == "ClientPayload") {
+                var clientPayload = utils.decodeClientPayload(v.delta.payload);
+                if (clientPayload && clientPayload.deltas) {
+                  for (var i in clientPayload.deltas) {
+                    var delta = clientPayload.deltas[i];
+                    if (delta.deltaMessageReaction) {
+                      delta.deltaMessageReaction.type = "message_reaction";
+                      delta.threadId = delta.threadKey.threadFbId;
```

Comment:
Not sure I follow 100% but if it seems like a value is sometimes in one place and sometimes in another, please support both 😄 
Sometimes people have different versions and it would be great to have the api work for everyone.